skyportal/handlers/api/photometry.py

```diff
@@ -1,13 +1,21 @@
 from astropy.table import Table
-import hashlib
 from marshmallow.exceptions import ValidationError
 import numpy as np
 import pandas as pd
 import sncosmo
 from sncosmo.photdata import PhotometricData
 from sqlalchemy.orm import joinedload
+from sqlalchemy import and_
 import uuid
 
+import math
+
+from sqlalchemy.ext.compiler import compiles
+from sqlalchemy.sql.expression import FromClause
+from sqlalchemy.sql import column
+
+import sqlalchemy as sa
+
```

Comment:
Going off of Ari's comments on readability/consistency, I would suggest reordering the imports here so that all the sqlalchemy imports are adjacent; `import math` and `import uuid` is at the top since those are standard library imports.

Reply:
done